From 34366fe222daf841b612cea3f7e89b683d3b48e9 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 8 Jun 2015 15:54:18 +0200
Subject: [PATCH 164/310] flow-tools: fix gcc5 build

gcc5 uses -std=gnu11 instead of -std=gnu89
by default

changed inline semantic leads to link errors:
 undefined reference to `eval_match_src_as'

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/flow-tools/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/flow-tools/Makefile b/net/flow-tools/Makefile
index 3f80d71..76e92cc 100644
--- a/net/flow-tools/Makefile
+++ b/net/flow-tools/Makefile
@@ -32,6 +32,8 @@ define Package/flow-tools/description
 	NetFlow data from Cisco and Juniper routers.
 endef
 
+TARGET_CFLAGS+=-std=gnu89
+
 define Build/Compile
 	$(call Build/Compile/Default,all)
 endef
-- 
2.5.0

